src/containers/SearchContainer/index.js

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { navigate } from '@reach/router';
+import { Box, Heading, Tabs, Tab, Text, TextInput } from 'grommet';
+import { Search as SearchIcon } from 'grommet-icons';
+
+import { Link, HighlightedText } from '../../components';
+import { useParams } from '../../hooks/use-params';
+
+const capitalize = text =>
+  text.substring(0, 1).toUpperCase() + text.substring(1);
+
+const categoryLabel = category =>
+  category === 'homepanels' ? 'Home' : capitalize(category);
+
+const Results = ({ searchTerm, results }) => {
+  if (results.length === 0) {
+    return (
+      <Text>
+        No results found for term <code>{searchTerm}</code>
+      </Text>
+    );
+  }
+
+  return results.map(({ doc, title, body, tags }, index) => (
+    <Box pad={{ bottom: 'medium' }} border={{ side: 'bottom' }} key={index}>
+      <Text size="small">{categoryLabel(doc.sourceInstanceName)}</Text>
+      <Link to={doc.path}>
+        <HighlightedText content={doc.title} positions={title} />

```

Comment:
I'm not sure it is clear to have both `title` and `doc.title`.

Reply:
Agree. title is actually the array of positions of the matches in the title. Similarly with tags and body. Would it be more clear if I just called them titlePos, or titlePositions etc?

Reply:
Yes, easier to deduce and connect its context

Reply:
Yes, easier to deduce and connect its context